packages/client/src/containers/Signup/Signup.container.js

```diff
@@ -26,12 +32,28 @@ function Signup() {
 
   const cleanUpValidation = () => {
     setFormErrors({
-      fullname: {},
-      email: {},
-      password: {},
+      fullname: '',
+      email: '',
+      password: '',
     });
   };
-
+  const addUserToDb = useCallback(async (userCreated, fullname) => {
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        full_name: fullname,
+        email: userCreated.email,
+        uid: userCreated?.uid,
+      }),
+    };
+    await fetch(`${apiURL()}/users`, requestOptions)
```

Comment:
Is there a particular reason for mixing await/async with `.then` below?

Reply:
removed